dotcom-rendering/src/components/NewsletterCard.tsx

```diff
@@ -143,31 +201,49 @@ export const NewsletterCard = ({
 			css={groupItemStyle}
 			aria-label={newsletter.name}
 		>
-			<NewsletterDetail text={newsletter.frequency} />
-			<h3>{newsletter.name}</h3>
-			<p>{newsletter.description}</p>
-			<div css={buttonHolderStyle}>
-				<Button
-					{...getButtonInitialAriaAttribute(newsletter.name)}
-					priority="tertiary"
-					size="xsmall"
-					iconSide="left"
-					icon={
-						<>
-							<span className={ICON_PLUS_CLASS}>
-								<SvgPlus />
-							</span>
-							<span className={ICON_TICK_CLASS}>
-								<SvgCheckmark />
-							</span>
-						</>
-					}
-					cssOverrides={buttonStyle}
-					data-newsletter-id={newsletter.identityName}
-					data-role={BUTTON_ROLE}
-				>
-					Sign up
-				</Button>
+			{!!newsletter.illustrationCard && (
+				<div css={illustrationStyle}>
+					<CardPicture
+						imageSize="carousel"
+						alt=""
```

Comment:
Can we pass some `alt` text here? e.g. `Newsletter ${newsletter.name}`. Doesn't have to be this, whatever you think it's best!

Reply:
The images will be purely decorative so the advice would be to have an empty `alt` so they are ignored by screen readers: https://www.w3.org/WAI/tutorials/images/decorative/